src/App.jsx

```diff
@@ -246,32 +233,15 @@ function App() {
         <NavigationControl position="top-right" />
       </MapGL>
 
-      <div className="slider-container">
-        {/* Agrega un botón o elemento para cambiar la visibilidad de Filtros */}
-        <Slider
-          max={months}
-          valueLabelDisplay="auto"
-          value={monthRange}
-          step={1}
-          getAriaValueText={valueLabelFormat}
-          valueLabelFormat={valueLabelFormat}
-          onChange={handleChange}
-          aria-labelledby="non-linear-slider"
-        />
-        <div id='referenciasFechas'>
-          <div>
-            <h6 id='fechaInicio'>
-              {dates.min.getMonth()}/{dates.min.getFullYear()}
-            </h6>
-          </div>
-          <div>  </div>
-          <div>
-            <h6 id='fechaCierre'>
-              {dates.max.getMonth()}/{dates.max.getFullYear()}
-            </h6>
-          </div>
-        </div>
-      </div>
+      <MonthsSlider
+          className="slider-container"
+          monthRange={monthRange}
+          setMonthRange={setMonthRange}
+          totalMonths={months}
+          startDateLabel="2/2020"
+          endDateLabel="9/2023"
```

Comment:
shouldn't be hardcoded

Reply:
Deberíamos crear una issue, lo muevo a un objeto en el top-level, o qué accionable puedo tomar?